### PR TITLE
Include additional temperatures for the transfer system and set TSup_…

### PR DESCRIPTION
**Pull request by larissakuehn**
The following changes have been made: 

- Add the temperature for the transfer system within PartialTransfer and set it to the supply temperature specified in the SystemParameters. 

- The original supply temperature now receives the maximum value of the required transfer temperature

### CODE REVIEW

File: BESMod/Systems/Hydraulical/BaseClasses/PartialHydraulicSystem.mo [hunk 1]
```diff
@@ -124,7 +124,8 @@ partial model PartialHydraulicSystem
       Placement(transformation(extent={{-12,-104},{90,28}})));
 
   replaceable BESMod.Systems.Hydraulical.Transfer.BaseClasses.PartialTransfer transfer(
-      dp_nominal=fill(0, transfer.nParallelDem))                                                                                                                                                                                                         constrainedby
+      dp_nominal=fill(0, transfer.nParallelDem), TTran_nominal=
```
Comment (FWuellhorst) [inline BESMod/Systems/Hydraulical/BaseClasses/PartialHydraulicSystem.mo:127]: `Tran` should be `Tra`, always three letters.

File: BESMod/Systems/Hydraulical/Transfer/BaseClasses/PartialTransfer.mo [hunk 2]
```diff
@@ -3,9 +3,14 @@ partial model PartialTransfer "Partial transfer model for BES"
   extends BESMod.Utilities.Icons.TransferIcon;
   extends
     BESMod.Systems.BaseClasses.PartialFluidSubsystemWithParameters(
+    TSup_nominal=fill(max(TTran_nominal),nParallelSup),
       dTTra_nominal={if TSup_nominal[i] > 64.9 + 273.15 then 15 elseif
         TSup_nominal[i] > 44.9 + 273.15 then 10 else 7 for i in 1:nParallelDem},
       m_flow_nominal=Q_flow_nominal ./ (dTTra_nominal .* 4184));
+
+  parameter Modelica.Units.SI.Temperature TTran_nominal[nParallelDem] "Nominal supply temperature to transfer systems"
```
Comment (FWuellhorst) [inline BESMod/Systems/Hydraulical/Transfer/BaseClasses/PartialTransfer.mo:11]: same

**Review by FWuellhorst — CHANGES_REQUESTED** [review]
Other than the naming, everything looks good! Thanks for noticing the error!

File: BESMod/Systems/Hydraulical/BaseClasses/PartialHydraulicSystem.mo (hunk 1 above)
Comment (larissakuehn) [reply BESMod/Systems/Hydraulical/BaseClasses/PartialHydraulicSystem.mo]: I changed the naming, thanks!

**Review by FWuellhorst — APPROVED** [review]

**Pull request merged**